.github/workflows/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Document how to upload offline tokens to GitHub secrets for loadtest workflow
+
+Staging users are splitted and stored as base64 encoded secrets - STAGING_USERS_1, STAGING_USERS_2, STAGING_USERS_3, STAGING_USERS_4
+The GITHUB secrets link is <https://github.com/redhat-appstudio/e2e-tests/settings/secrets/actions>
+or for any fork (like: naftalysh) in <https://github.com/naftalysh/e2e-tests/settings/secrets/actions>
+
+# The users secrets creation process
+
+* Input file - (ex: users_approved.list) space separated file of users records with structure as below:
+  username (test-rhtap-1,..test-rhtap-100) password offline_token ssourl apiurl true
+
+Prerequisite: that file is populated as part of Jan's process (<https://gitlab.cee.redhat.com/jhutar/mystoneinst/-/blob/main/README.md#get-offline-tokenpy>)
+
+  a. get a space separated file like the input file above with just the two fields - username and password
```

Comment:
I would not duplicate that process here. Link above is sufficient.

Reply:
I will deprecate it